File: code/TemplateStudioForUWP/Templates/_comp/_shr/Page.xUnit._VB/Param_ProjectName.Tests.xUnit/Tests$wts.ItemName_gpostaction.vb
'{[{
Imports Param_RootNamespace.ViewModels
'}]}

Public Class Tests
    '^^
    '{[{

    ' TODO: Add tests for functionality you add to wts.ItemNameViewModel.
    <Fact>
    Public Sub Testwts.ItemNameViewModelCreation()
        ' This test is trivial. Add your own tests for the logic you add to the ViewModel.
        Dim vm = new wts.ItemNameViewModel()
        Assert.NotNull(vm)
    End Sub
    '}]}
End Class

File: code/TemplateStudioForUWP/Templates/_comp/_shr/Page.xUnit/Param_ProjectName.Tests.xUnit/Tests$wts.ItemName_gpostaction.cs
//{[{
using Param_RootNamespace.ViewModels;
//}]}

namespace Param_RootNamespace.Tests.XUnit
{
    public class Tests
    {
        //^^
        //{[{

        // TODO: Add tests for functionality you add to wts.ItemNameViewModel.
        [Fact]
        public void Testwts.ItemNameViewModelCreation()
        {
            // This test is trivial. Add your own tests for the logic you add to the ViewModel.
            var vm = new wts.ItemNameViewModel();
            Assert.NotNull(vm);
        }
        //}]}
    }
}
